shared/react-native/android/app/src/main/java/io/keybase/ossifrage/modules/IntentHandler.java

```diff
@@ -163,15 +167,14 @@ public String getName() {
     }
 
     @ReactMethod
-    public void handlePushNotification(String convID, String payload, Integer membersType,
```

Comment:
Why does this get deleted, was it unused?

Reply:
Yep, unused.